### ISSUE TITLE
Support to hide the connected pop-up window

### ISSUE THREAD

**Issue by cweijan** [rustdesk/rustdesk#1744]
When I connect, a popup will be displayed, since I am using it in a local area network, I can be sure that rustdesk is secure, can add an option to hide the popup?
![image](https://user-images.githubusercontent.com/27798227/196329228-a2f989b2-6267-4bb0-8c82-3c56c8d5ce02.png)

Thanks for this project, it's a good vnc replacement.

**Comment by rustdesk** [comment]
Let's see if we can

**Comment by rustdesk** [comment]
Let's see how many guys have such requirement first.

**Comment by rustdesk** [comment]
Please tell your reason.

**Comment by bmcgonag** [comment]
If this is about hiding the pop-up on the client side, I would personally not want that hidden as a client.  this is a security risk in my opinion.  You can minimize the pop-up when you connect as it is.  Not sure there's any need to have this hidden by default.  Just my 2 cents.

**Comment by chmis8000** [comment]
This aspect is better at ToDesk.
 Can be hidden to the right to show only a small click area without taking up the taskbar.
At present, the style of rustdesk is very similar to that of AnyDesk. Too many windows pop up, occupying the taskbar.

**Comment by 53b4** [comment]
I would welcome this  feature also.

**Comment by Manedic** [comment]
se requiere para administrar y monitorear los equipos de una red interna

**Comment by luongconghao** [comment]
> Please tell your scenario and reason.

I really need this feature. My company has its own server. and i want to have "hidden remote" or "silient remote" feature. This option can set in server for all client or set client side for some client . if can't support this feature. can you tell me where to comment out the code, i want to rebuild the client

**Comment by rustdesk** [comment]
@Heap-Hop will add options as like AnyDesk.

**Comment by rustdesk** [comment]
@Heap-Hop notice the chat button on controlling side if not pop up.

**Comment by rustdesk** [comment]
Some vendor decide if pop up when connecting from different account.

**Comment by ylluminate** [comment]
TeamViewer facilitates this and it is very useful for our scenario. TeamViewer has a superior model in nearly every aspect than AnyDesk for emulation.

**Comment by ggibby** [comment]
The pop-up takes screen space and attention when I really just want to get work done.
Closing it kills the connection, and minimizing takes space on the task bar.

**Comment by ZaxonXP** [comment]
Can we have a configuration option and let user decide if this pop up should occur when remote party connect to the machine? Personally I would prefer to have an extra icon on the systray instead of this pop up.

I saw this closed after the code got merged. How to configure the client in order to make this work? I tried to look over all the settings and still did not manage to set it without the pop up.
